index.html

```diff
@@ -26,17 +26,22 @@
             </div>
         </nav>
         <pagination total-items="forksCount" items-per-page="perPage" ng-model="currentPage" ng-change="getForks()" 
-        max-size="maxSize" class="pagination-sm" boundary-links="true" rotate="false"></pagination>
-        <div class="alert alert-danger" role="alert" ng-show="noRepo">Sorry, repository not found!</div>
-        <div class="alert alert-danger" role="alert" ng-show="noFork">Sorry, nobody gave a fork!</div>
-        <article class="panel panel-primary" ng-hide="noRepo || noFork">
-            <div class="panel-heading">Last forks <span class="label label-success pull-right">Total forks: {{ forksCount }}</span></div>
-            <ul class="list-group" ng-repeat="fork in forks">
-                <li class="list-group-item">
-                    <a ng-href=" {{ fork.owner.html_url }}">{{ fork.owner.login }}</a>
-                    <span class="pull-right">{{ fork.created_at | date }}</span>
-                </li>
-            </ul>
+        max-size="maxSize" class="pagination-sm" boundary-links="true" rotate="false" ng-hide="showMessage"></pagination>
+        <div class="alert alert-danger" role="alert" ng-show="showMessage"> {{ message }}</div>
+        <article class="panel panel-primary" ng-hide="showMessage">
+            <div class="panel-heading">Total forks: {{ forksCount }}</span></div>
+            <table class="table table-striped table-hover">
+            <tr>
+                <th>#</th>
+                <th>User</th>
+                <th>Date forked</th>
+            </tr>
+                <tr ng-repeat="fork in forks">
+                    <td>{{ forks.indexOf(fork) + 1}}
```

Comment:
Nice for the fork number, I was thinking of having here the actual fork number with
{{ forksCount - (currentPage - 1) * perPage + forks.indexOf(fork) }}
So that in the first page, the number for the first fork is equal to the total number of forks, and then decrements... to the last fork with number 1.
In this case we would not need anymore the panel and the panel heading, just a table

One more thing, I was thinking of adding one more column with header Fork and a link to the forked repository.